platform-tests/acceptance/aws_endpoint_check_test.go

```diff
@@ -0,0 +1,130 @@
+package acceptance_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+
+	"github.com/cloudfoundry/cf-test-helpers/cf"
+	"github.com/cloudfoundry/cf-test-helpers/generator"
+)
+
+type instanceIdentity struct {
+	Region string `json:"region"`
+}
+
+func GetAWSRegion() (string, error) {
+	resp, err := http.Get("http://169.254.169.254/latest/dynamic/instance-identity/document")
```

Comment:
Have you seen IMDSv2? What's your opinion, should we be using it? Or are we confident enough in our existing security posture to continue using IMDSv1?

Reply:
I hadn't seen IMDSv2. I can understand peoples concerns about the security of IMDS. To be honest I don't like this getting the region from IMDS. I think it should come from the tests config. It currently isn't there. Let me see how hard it would be to add it.

Reply:
Changed to use the environment variable.